### PR TITLE
Move etcd to graffiti

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run scale-tests

**Comment by lebauce** [comment]
run scale-tests

File: analyzer/server.go [hunk 1]
```diff
@@ -121,27 +125,43 @@ func (s *Server) createStartupCapture(ch *api.CaptureAPIHandler) error {
...
 	}
 
-	if err := s.httpServer.Listen(); err != nil {
+	if err := s.hub.Start(); err != nil {
+		return err
+	}
+
+	s.etcdClient.Start()
+
+	if s.graphStorage != nil {
```
Comment (safchain) [inline analyzer/server.go]: I wonder if it is not better to start storage before http.Listen like before in order to be sure that we won't have pod -> hub message while not having the storage ready ?

**Comment by lebauce** [comment]
run functional-tests-backend-orientdb

**Pull request merged**